domains/POAS/events/lpa-access-granted/examples/example.json

```diff
@@ -0,0 +1,8 @@
+{
+  "uid": "M-1234-5678-9012",
+  "lpaType": "personal-welfare",
+  "actors" : {
+      "9ac5cb7c-fc75-40c7-8e53-059f36dbbe3d": "urn:fdc:gov.uk:2022:XXXX-XXXXXX",
+      "eda719db-8880-4dda-8c5d-bb9ea12c236f": null
```

Comment:
Presumably the null option handles paper channel actors who have never signed onto MRLPA? I wonder if it's even worth sending them as we can't do anything with them Use side since they have no subject identifier.

Reply:
Unless we have a use for them, feels like we shouldn't be sending null users that can't be granted access at the other end.